pkg/instrumentation/nodejs.go

```diff
@@ -22,7 +22,7 @@ import (
 
 const (
 	envNodeOptions          = "NODE_OPTIONS"
-	nodeRequireArgument     = " --require /otel-auto-instrumentation-nodejs/autoinstrumentation.js"
+	nodeRequireArgument     = " --require /otel-auto-instrumentation-nodejs/node_modules/@opentelemetry/auto-instrumentations-node/build/src/register.js"
```

Comment:
Would this work? That's the entrypoint we provide for these kinds of things.
```suggestion
	nodeRequireArgument     = " --require /otel-auto-instrumentation-nodejs/node_modules/@opentelemetry/auto-instrumentations-node/register"
```

Reply:
I tried but i got

Error: Cannot find module '/otel-auto-instrumentation-nodejs/node_modules/@opentelemetry/auto-instrumentations-node/register'